### PR TITLE
[CCDB-4843] Register TimestampConverter in DebeziumLogicalConverters behind a config for BigQuerySink Connector

### PR DESCRIPTION
**Pull request by sp-gupta**
…behind a config for BigQuerySink Connector

Refer to [this](https://confluentinc.atlassian.net/browse/CCDB-4843) ticket for details related to testing and Release Plan

### CODE REVIEW

File: kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java [hunk 1]
```diff
@@ -546,6 +554,12 @@ public static ConfigDef getConfig() {
             BIGQUERY_CLUSTERING_FIELD_NAMES_VALIDATOR,
             BIGQUERY_CLUSTERING_FIELD_NAMES_IMPORTANCE,
             BIGQUERY_CLUSTERING_FIELD_NAMES_DOC
+        ).define(
```
Comment (b-goyal) [inline kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java:557]: This should be an internal config.

File: kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/convert/BigQueryRecordConverter.java [hunk 2]
```diff
@@ -236,6 +238,10 @@ private Object convertLogical(Object kafkaConnectObject,
                                 Schema kafkaConnectSchema) {
     LogicalTypeConverter converter =
         LogicalConverterRegistry.getConverter(kafkaConnectSchema.name());
+
+    if(shouldConvertDebeziumTimestampToInteger && converter instanceof DebeziumLogicalConverters.TimestampConverter) {
+      return (Long) kafkaConnectObject;
```
Comment (b-goyal) [inline kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/convert/BigQueryRecordConverter.java:243]: is the kafka schema type of this object always INT64 ?

**Review by b-goyal — CHANGES_REQUESTED** [review]
Don't think we need to target 1.6.x,  let's target customer's current version.

**Comment by sp-gupta** [comment]
> Don't think we need to target 1.6.x, let's target customer's current version.

I think this issue can happen for a customer in CP with the same configs in future. I was thinking, we make this change once and for all.

File: kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java (hunk 1 above)
Comment (sp-gupta) [reply kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java:557]: Sure, done

File: kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/convert/BigQueryRecordConverter.java (hunk 2 above)
Comment (sp-gupta) [reply kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/convert/BigQueryRecordConverter.java:243]: As per the [current](https://github.com/confluentinc/kafka-connect-bigquery/blob/39a43f9295d12b543e6a3ebe8f49c80e562e7087/kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/convert/logicaltype/DebeziumLogicalConverters.java#L175) conversion of Timestamp object to String, it assumes the object to be of INT64 only

**Comment by b-goyal** [comment]
> 

Thought it got introduced in 2.3.x, if it is there in 1.6.x, we should target 1.6.x

**Review by b-goyal — APPROVED** [review]
Thanks @sp-gupta LGTM

**Pull request merged**